Frends.AzureBlobStorage.CreateContainer/Frends.AzureBlobStorage.CreateContainer.Tests/.env.example

```
@@ -0,0 +1,4 @@
+Frends_AzureBlobStorage_ConnString="X"
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The Frends_AzureBlobStorage_ConnString key should be in uppercase

(LowercaseKey)